tests/cli/vyper_json/test_output_dict.py

```diff
@@ -38,7 +38,7 @@ def test_keys():
                 "breakpoints": data["breakpoints"],
                 "pcBreakpoints": data["pc_breakpoints"],
                 "sourceMap": data["source_map"]["pc_pos_map_compressed"],
-                "pcPosMap": data["source_map"]["pc_pos_map"],
+                "sourceMapFull": data["source_map"],
```

Comment:
nice. i think we can remove the other new fields now, since they are contained within sourceMapFull.

Reply:
do we want the other new fields to be sorted?